RELEASE.md

```diff
@@ -51,10 +51,11 @@ A number of files have to be re-generated, this is automated with the following
 $ make clean generate
 ```
 
-Bump the version of the `pkg/apis/monitoring` package in `go.mod`:
+Bump the version of the `pkg/apis/monitoring` and `pkg/client` packages in `go.mod`:
```

Comment:
we need also to bump `github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring` in `pkg/client/go.mod`?